pybricksdev/ble/pybricks.py

```diff
@@ -328,6 +328,12 @@ def _standard_uuid(short: int) -> str:
 .. availability:: Since Pybricks protocol v1.0.0.
 """
 
+DVC_NAME_UUID = _standard_uuid(0x2A00)
+"""Standard Device Name UUID
+
+.. availability:: Since Pybricks protocol v1.0.0.
```

Comment:
It seems like this would have been around since the beginning but I wanted to double-check that with you.

Reply:
Yes, this is a required characteristic for all BLE devices.